zipkin-simple.js

```diff
@@ -127,6 +127,10 @@ function sendTrace (trace, data) {
 			timestamp: time,
 			value: data.annotations[i]
 		})
+
+		if (data.annotations[i] === "cr") {
+			body.duration = time - trace.timestamp
```

Comment:
the timestamp of the start time is already added, the timestamp of the end is added a the timestamp of the cr annotation (on line 127).

Is this the correct way for zipkin or should I also update the span timestamp?